### ISSUE TITLE
Default file is not loaded at start

### ISSUE THREAD

**Issue by monolifed** [billthefarmer/editor#12]
Default file "Editor.txt" is set but not loaded at start.

**Comment by billthefarmer** [comment]
This is true, O wise one. I hadn't thought of that because the default file is where text entered without first loading a file goes. That will be in the next version.
